### PR TITLE
Reset root password

### LINKED ISSUE #322: Reset root password

### PR DESCRIPTION
**Pull request by SirCmpwn**
Closes #322

### CODE REVIEW

**Comment by na3d** [comment]
Are there any tests that should be moved over into PasswordInput.spec.js?

**Comment by SirCmpwn** [comment]
No, it seems like there aren't any tests for the password input in general. I'll file this as a bug, since the password input wasn't really introduced in this PR.

**Comment by na3d** [comment]
Sounds good 👍

File: src/linodes/layouts/RepairPage.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+        <h2>
+          Reset root password
+          <HelpButton to="http://example.org" />
+        </h2>
+        {body}
+      </div>
+    );
+  }
+
+  renderRebuild() {
```
Comment (eatonphil) [inline src/linodes/layouts/RepairPage.js]: You can drop this - it is supposed to be in its own tab.

Comment (na3d) [inline src/linodes/layouts/RepairPage.js]: Should I create that tab for the stubbing of things?

**Comment by eatonphil** [comment]
Looks good.

File: src/linodes/layouts/RepairPage.js (hunk 1 above)
Comment (SirCmpwn) [inline src/linodes/layouts/RepairPage.js]: :+1:

**Comment by SirCmpwn** [comment]
Merge away.

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**